src/users/services/jurisdiction.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { JurisdictionService } from './jurisdiction.service';
+
+describe('Jurisdiction service', () => {
+    const mockedHttpClient = jasmine.createSpyObj('mockedHttpClient', ['get']);
```

Comment:
Shouldn't this be spying on HttpClient instead of itself. However, if it works it works. :)